scripts/download.sh

```diff
@@ -34,6 +34,8 @@ else
   echo "Snapshot already present"
 fi
 
-if [ ! -z "$STATS_IP" ]; then
-  echo -n "ctlstore.reflector.init_snapshot_download_time:$(($END - $START))|h|#$TAGS" | nc -u -w1 $NODE_IP $STATS_PORT
-fi
\ No newline at end of file
+if [ -z "$STATS_IP" ]; then
+  METRICS="/var/spool/ctlstore/metrics.json"
+  echo "{\"startTime\": $(($END - $START)), \"downloaded\": true}" > $METRICS
```

Comment:
true needs to be set dynamically depending on whether or not the ldb was actually downloaded